domains/amc.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "It is a portfolio",
+    "repo": "https://github.com/AdrianMartinCano/portfolio/",
+    
+    "owner": {
+        "username": "AdrianMartinCano",
+        "email": "amc.1994.mca@gmail.com",
+    },
+    "record": {
+        "CNAME": " adrianmartincano.github.io/portfolio/"
```

CNAME should not contain `/`
```suggestion
        "CNAME": "adrianmartincano.github.io"
```